src/Goerli-DssSpell.sol

```diff
@@ -89,58 +49,93 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
 
-        // Configure TeleportRouter
-        TeleportRouterLike(ROUTER).file("gateway", DOMAIN_STA, TELEPORT_GATEWAY_STA);
+    function _updateDoc(bytes32 ilk, string memory doc) internal {
+        ( , address pip, uint48 tau, ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+        require(pip != address(0), "DssSpell/unexisting-rwa-ilk");
 
-        // Authorize TeleportGateway to use the escrow
-        EscrowLike(ESCROW_STA).approve(DAI, TELEPORT_GATEWAY_STA, type(uint256).max);
-
-        // Configure Chainlog
-        DssExecLib.setChangelogAddress("STARKNET_TELEPORT_BRIDGE", TELEPORT_GATEWAY_STA);
-        DssExecLib.setChangelogAddress("STARKNET_TELEPORT_FEE", LINEAR_FEE_STA);
-
-        DssExecLib.setChangelogVersion("1.14.4");
-
-        // ------------------ MOMC Parameter Changes -----------------
-        // https://vote.makerdao.com/polling/QmahDuNx#poll-detail
-
-        // Increase the MANA-A Stability Fee from 4.5% to 7.5%
-        DssExecLib.setIlkStabilityFee("MANA-A", SEVEN_PT_FIVE_PERCENT_RATE, true);
+        // Init the RwaLiquidationOracle to reset the doc
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, // ilk to update
+            0,          // price ignored if init() has already been called
+            doc, // new legal document
+            tau         // old tau value
+        );
+    }
 
-        // Decrease the MANA-A line from 17 million DAI to 10 million DAI
-        DssExecLib.setIlkAutoLineDebtCeiling("MANA-A", 10 * MILLION);
+    function actions() public override {
+        // -------------------- Update RWA007 Legal Documents ---------------------
+        // TODO: Add forum post link
+        //
+        // Monetalis (RWA007-A) legal update doc
+        _updateDoc("RWA007-A", "QmejL1CKKN5vCwp9QD1gebnnAM2MJSt9XbF64uy4ptkJtR");
+
+        // -------------------- Update RWA008 Legal Documents ---------------------
+        // TODO: Add forum post link
+        //
+        // SG Forge OFH (RWA008-A) legal update doc
+        _updateDoc("RWA008-A", "QmZ4heYjptvj3ovafADJpXYMFXMyY3yQjkTXpvjFPnAKcy");
+
+        // -------------------- Update RWA009 Legal Documents ---------------------
+        // TODO: Add forum post link
+        //
+        // HVB (RWA009-A) legal update doc
+        _updateDoc("RWA009-A", "QmeRrbDF8MVPQfNe83gWf2qV48jApVigm1WyjEtDXCZ5rT");
+
+        // RWA007-A autoline changes:
+        // - increase DC to 500m
+        // - increase autoline `gap` to 100m
+        //
+        // https://vote.makerdao.com/polling/QmSfMtTM#poll-detail
+        DssExecLib.setIlkAutoLineParameters(
+            "RWA007-A",
+            500 * MILLION,
+            100 * MILLION,
+            1 weeks
+        );
+
+        // RETH-A autoline changes:
+        // - line 5m
+        // - gap 3m
+        // - ttl 8h
+        //
+        //  https://vote.makerdao.com/polling/QmfMswF2#poll-detail
+        DssExecLib.setIlkAutoLineParameters(
+            "RETH-A",
```

Comment:
reminder: coordinate with Oracle CU to ensure MegaPoker updates related to `RETH-A` are deployed on mainnet before the mainnet exec goes live (on goerli OmegaPoker `refresh` and `poke` can be called once the spell is cast, to check with TechOps eventually).